packages/api/lambdas/internal-reconciliation-report.js

```diff
@@ -48,10 +54,19 @@ async function internalRecReportForCollections(recReportParams) {
     { ...searchParams, sort_key: ['name', 'version'] }, 'collection', process.env.ES_INDEX
   );
 
+  const collectionPgModel = new CollectionPgModel();
+  const knex = await getKnexClient();
+
   // get collections from database and sort them, since the scan result is not ordered
-  const dbSearchParams = convertToDBCollectionSearchParams(recReportParams);
-  const dbCollectionsQueue = (new Collection()).search(dbSearchParams);
-  const dbCollectionsSearched = await dbCollectionsQueue.empty();
+  const [updatedAtRangeParams, dbSearchParams] = convertToDBCollectionSearchObject(recReportParams);
+
+  const dbCollectionsSearched = await collectionPgModel.searchWithUpdatedAtRange(
+    knex,
+    dbSearchParams,
+    updatedAtRangeParams
+  );
+
+  // TODO: This should work, but will return *postgres* collections.
```

Comment:
I'm not sure what the TODO is here or if it was already addressed?

Reply:
It was already addressed.      01d0ac122